### ISSUE TITLE
Zoom to last navigable piece

### ISSUE THREAD

**Issue by Dimitar5555** [Zaczero/osm-relatify#69]
When a gap forms in a relation, Relatify says that "Some ways are not used". It still shows animated line from the start to the gap (so it knows where the gap is) but as an end user, you can't jump to that last navigable segment.

Even without gaps, such feature could be useful, when "debugging" spaghetti lines.

**Comment by Zaczero** [comment]
I agree it would be nice to automatically jump to the last known segment. Thank you for the great idea! :slightly_smiling_face:
